matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/EventRelationsAggregationProcessor.kt

```diff
@@ -316,14 +316,16 @@ internal class EventRelationsAggregationProcessor @Inject constructor(
             ContentMapper
                     .map(eventAnnotationsSummaryEntity.pollResponseSummary?.aggregatedContent)
                     ?.toModel<PollSummaryContent>()
-                    ?.apply {
-                        totalVotes = 0
-                        winnerVoteCount = 0
-                        votes = emptyList()
-                        votesSummary = emptyMap()
-                    }
-                    ?.apply {
-                        eventAnnotationsSummaryEntity.pollResponseSummary?.aggregatedContent = ContentMapper.map(toContent())
+                    ?.let { existingPollSummaryContent ->
+                        eventAnnotationsSummaryEntity.pollResponseSummary?.aggregatedContent = ContentMapper.map(
+                                PollSummaryContent(
+                                        myVote = existingPollSummaryContent.myVote,
```

Comment:
This code is quite weird to men the goal is to only keep `myVote`? CC @onurays

Reply:
Yes, it is to help reach myVote easily while rendering it.

Reply:
OK, thanks

Reply:
not for this PR, might be better suited as a separate type if possible :thinking:

Reply:
Can you clarify? You mean having something like `data class MutablePollSummaryContent` with `var` and mutable types and a `toPollSummaryContent()` method, or is it something else?

Reply:
I meant a dedicated `MyVotePollContent(myVote)`, avoiding the need for the other values to be set to default values